circle.yml

```diff
@@ -0,0 +1,16 @@
+dependencies:
+  override:
+    - yarn
+  pre:
+    - rm -rf ~/.yarn
+    - npm install -g yarn
```

Comment:
Ok, I'm clearly too tired for this. Setting node's version on its own does it with yarn 0.21.3, so I've removed the reinstall of yarn as suggested.